src/main/webapp/assets/tpl/apps/param.html

```diff
@@ -34,6 +34,10 @@
         <input type="checkbox" class="param-value">
         <% } %>
     </td>
+    <% } else if (sensitive == true && type == 'java.lang.String') { %>
+    <td>
+        <input type="password" class="param-value">
```

Comment:
What about the default value? And maybe worth using the `input-medium` class for consistency with the case below.